src/molecules/FeedbackPage/index.tsx

```diff
@@ -0,0 +1,167 @@
+import React, { useState } from "react";
+import styles from "./index.module.css";
+import Typography from "@mui/material/Typography";
+import { Box } from "@mui/material";
+import Rating from "@mui/material/Rating";
+import Button from "@mui/material/Button";
+import config from "./config.json";
+import { toast } from "react-hot-toast";
+
+const FeedbackPage: React.FC = () => {
+  const [star, setStar] = useState(1);
+  const [review, setReview] = useState("");
+
+  const handleFeedback = () => {
+    const rateBox = config.component.ratingBox;
+    const reviewContainer = config.component.reviewBox;
+
+    if (rateBox && reviewContainer) {
```

Comment:
there is lots of if else condition which you can reduce a lot like you can do it something like
`
const handleFeedback = () => {
  const rateBox = config.component.ratingBox;
  const reviewContainer = config.component.reviewBox;

  const sendReviewSuccess = () => {
    setTimeout(() => {
      toast.success(`Review sent successfully`);
      setReview("");
    }, 2000);
  };

  const sendReviewError = () => {
    toast.error(`Please provide valid review`);
  };

  if (rateBox && reviewContainer) {
    star === 0 ? sendReviewError() : sendReviewSuccess();
  } else if (rateBox && !reviewContainer) {
    star === 0 ? sendReviewError() : sendReviewSuccess();
  } else if (!rateBox && reviewContainer) {
    review === "" ? sendReviewError() : sendReviewSuccess();
  }
};

`